content/en/service_management/events/_index.md

```diff
@@ -21,11 +21,11 @@ further_reading:
   text: "Quick Start Guide"
 ---
 
-{{< site-region region="us3,us5,gov" >}}
-<div class="alert alert-warning">Event Management is not supported for this site.</div>
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Event Management beta is not supported for this site.</div>
```

Comment:
So—I think you're trying to say that it's the beta that's not supported for this site. However, if the beta isn't supported then the feature itself (beta or not) isn't supported either. I'm going to remove the word "beta."